src/ServiceInsight/ServiceControl/IServiceControl.cs

```diff
@@ -34,5 +34,8 @@ public class ServiceControlHeaders
     {
         public const string ParticularVersion = "X-Particular-Version";
         public const string TotalCount = "Total-Count";
+        public const string Link = "Link";
+        public const string PageSize = "Page-Size";
+        public const string PageNumber = "Page-Number";
```

Comment:
This is not yet implemented in SC. I added this and then forgot to remove when I realized we need to first introduce it to SC to make the logic in SI simpler.